src/shared/ui/survey/SimpleTextInput.tsx

```diff
@@ -0,0 +1,55 @@
+import { FC, useState } from 'react';
+import { TextInputProps } from 'react-native';
+
+import { useTranslation } from 'react-i18next';
+
+import { colors } from '@shared/lib';
+import { Input } from '@shared/ui';
+
+type ValueType = '' | 'http://www.w3.org/2001/XMLSchema#integer';
+
+type Config = {
+  isOptionalText: boolean;
+  maxLength: number;
+  removeBackOption: boolean;
+  valueType: ValueType;
+};
+
+type Props = {
+  valueType?: ValueType;
+  config: Config;
```

Comment:
Please put all props on the same level. No need to put them in the `config` object in this case